core/templates/pages/skill-editor-page/navbar/skill-editor-navbar.directive.ts

```diff
@@ -16,10 +16,11 @@
  * @fileoverview Directive for the navbar of the skill editor.
  */
 
+require('components/common-layout-directives/common-elements/' +
+  'confirm-or-cancel-modal.controller.ts');
 require(
   'components/common-layout-directives/common-elements/' +
   'loading-dots.directive.ts');
-
```

Comment:
Undo this change (We need a newline between service and directive imports.)

Reply:
Done.